tests/Formatter/ClassNameFormatterTest.php

```diff
@@ -54,11 +54,8 @@ public function testCompleteContextOnFailureWithExceptionInfo()
         $exception = new UserAlreadyExistsException("foo bar baz");
         $this->assertEquals(
```

Comment:
Should be test what happens when the currentContext/commandContext has the key 'exception' set (to for example null) and ensure it is overridden?